bucket/nats-server.json

```diff
@@ -17,7 +22,15 @@
             "64bit": {
                 "url": "https://github.com/nats-io/nats-server/releases/download/v$version/nats-server-v$version-windows-amd64.zip",
                 "extract_dir": "nats-server-v$version-windows-amd64"
+            },
+            "32bit": {
+                "url": "https://github.com/nats-io/nats-server/releases/download/v$version/nats-server-v$version-windows-386.zip",
+                "extract_dir": "nats-server-v$version-windows-386"
             }
+        },
+        "hash": {
+            "url": "https://github.com/nats-io/nats-server/releases/download/v$version/SHA256SUMS",
+            "find": "SHA256\\($basename\\)=\\s+([a-fA-F\\d]{64})"
```

Comment:
I have a feeling that 64 shouldn't be there at the end?

Reply:
The `find` field can be removed here entirely

Reply:
Also, you should update the manifest to 2.9.2 now

Reply:
I've removed the find field. And I have already changed it to be 2.9.2